routes/licenses.js

```diff
@@ -14,12 +14,36 @@ const mockResponse = [
   },
 ];
 
+routes.push({
+  path: '/licenses/compatible/{license}',
+  method: 'GET',
+  options: {
+    description: 'Compatible licenses',
+    notes: 'Returns a list of licenses that are compatible to the passed license',
+    validate: {
+      params: {
+        license: Joi.string(),
+      },
+    },
+    response: {
+      schema: Joi.array().items(licenseSchema),
+    },
+  },
+  handler: async (request, h) => {
+    const licenseStore = request.server.app.services.licenses;
```

Comment:
Not necessarily part of this PR, but once we come to moving the "business logic entry points" into controllers, we probably want to dependency inject the `LicenseStore` 🤔 So that we only have a single one for the whole app...?

Reply:
True! Probably it makes sense to pass the `request` reference to the yet-to-be-implemented "controller" function. For the sake of consistency we should find da solution that fits all routes. I'd say we address that issue when we implement the controller abstraction for all routes (ticket #16)